app/models/labware_creators/pooled_tubes_base.rb

```diff
@@ -93,10 +96,10 @@ def metadata_stock_barcode
     end
 
     def parent_metadata
-      if parent.is_a? Limber::Plate
```

Comment:
There is the concept of a plate meant to be used in Limber and one only meant to be used in Sequencescape. You get a warning in the GUI if you scan a non-Limber plate in Limber. 
Is this Limber::Plate check to do with that? And why do you replace it here but then use it in the support_v2_plate_source_plate code above?

Reply:
Maybe the method `source_plate` is badly named.  I took that from code James Glover wrote for cardinal pools plate labware creator to convert a `parent` into a V2 plate.  Not all the labware creators deal with plates I believe since one of the tests fails if I don't check that we have a plate here.  In which case it falls back to passing the parent's barcode to the LabwareMetadata.  I guess when it's a tube, as it is in one of the tests, it gives the barcode of the tube to `LabwareMetadata` and the labware gets looked up that way.  I don't know if there are any real life instances where the labware would not be able to be looked up by its barcode.  I assumed so because of the dual functionality constructor that can take the labware itself or the barcode.  But perhaps that's legacy functionality and we could make our lives a lot easier by only using the barcodes all the time?

Reply:
I think I've change this now to be cleaner.